lib/tasks/recalculate_status_after_sales_over_retirement_age_validation.rake

```diff
@@ -0,0 +1,20 @@
+desc "Recalculates status for 2024 logs that will trigger new sales over retirement age soft validation"
+task recalculate_status_over_retirement: :environment do
+  validation_trigger_condition = "(ecstat1 != 5 AND age1 > 66) OR (ecstat2 != 5 AND age2 > 66) OR (ecstat3 != 5 AND age3 > 66) OR (ecstat4 != 5 AND age4 > 66) OR (ecstat5 != 5 AND age5 > 66) OR (ecstat6 != 5 AND age6 > 66)"
+  SalesLog.filter_by_year(2024).where(status: "pending", status_cache: "completed").where(validation_trigger_condition).find_each do |log|
```

Comment:
I think we do want to do this for pending logs, because we don't recalculate the status when unpending (we just use the cache). Of course this might also catch a load of logs that will never be unpended, but there's no easy way to avoid that.